src/CMakeLists.txt

```diff
@@ -15,20 +15,22 @@
 #===============================================================================
 
 set(TARGET_NAME ${LIB_NAME})
+
 file(GLOB_RECURSE HEADERS
-    ${CMAKE_SOURCE_DIR}/include/*.h
-    ${CMAKE_SOURCE_DIR}/include/*.hpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/../include/*.h
+    ${CMAKE_CURRENT_SOURCE_DIR}/../include/*.hpp
     )
 file(GLOB_RECURSE SOURCES
-    ${CMAKE_SOURCE_DIR}/src/*.c
-    ${CMAKE_SOURCE_DIR}/src/*.cpp
-    ${CMAKE_SOURCE_DIR}/src/*.h
-    ${CMAKE_SOURCE_DIR}/src/*.hpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/*.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/*.h
+    ${CMAKE_CURRENT_SOURCE_DIR}/*.hpp
     )
 include_directories(
-    ${CMAKE_SOURCE_DIR}/src
-    ${CMAKE_SOURCE_DIR}/src/common
-    ${CMAKE_SOURCE_DIR}/src/cpu/xbyak
+    ${CMAKE_CURRENT_SOURCE_DIR}/..
```

Comment:
Is there any specific reason for adding this line?
`${CMAKE_CURRENT_SOURCE_DIR}/..`